pkgs/desktops/gnome/extensions/system-monitor/default.nix

```diff
@@ -17,6 +17,11 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
+    # GNOME 43 compatibility
+    (fetchpatch {
+      url = "https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet/pull/754/commits/3b3a617f78c5e9bbce0aa2864b3989335edb37b7.patch";
```

Comment:
Won't this URL be garbage collected (breaking long-term reproducibility)?

Reply:
I don’t see any reason to think that. https://github.com/NixOS/nixpkgs/pull/1/commits/d0d4123962ed53266319dff6b19c42444b26bb7a.patch stlil works.

Reply:
That's good to know. I wonder if that holds if a PR is rebased before merging though? Does github really keep every revision, even "unreferenced" ones?

Reply:
As far as I know, by default yes.

Reply:
They keep them long enough that fetching patches like this was not a major problem yet unless the project is completely deleted.